src/backend/base/langflow/components/langchain_utilities/langchain_hub.py

```diff
@@ -99,9 +99,11 @@ async def build_prompt(
         template = self._fetch_langchain_hub_template()
 
         # Get the parameters from the attributes
-        original_params = {k.removeprefix("param_"): v for k, v in self._attributes.items()}
+        params_dict = {param: getattr(self, "param_" + param, f"{{{param}}}") for param in template.input_variables}
+        original_params = {k: v.text if hasattr(v, "text") else v for k, v in params_dict.items()}
```

Comment:
or this, by adding `None` as default in the `getattr` call
```suggestion
        original_params = {k: v.text if hasattr(v, "text") else v for k, v in params_dict.items() if v is not None}
```

Reply:
@ogabrielluiz the reason for this one is because the input can be a plain string (then it wouldnt have the attribute - say, if the user just typed it). OR it can be a message, if it was piped in from another component. Does that make sense or maybe there's another way to handle this?

Reply:
PS: Thanks so much for the feedback when i know you're super swamped :)